asserts/model.go

```diff
@@ -707,6 +732,20 @@ func assembleModel(assert assertionBase) (Assertion, error) {
 			grade = ModelGrade(gradeStr)
 		}
 
+		storageSafetyStr, err := checkOptionalString(assert.headers, "storage-safety")
+		if err != nil {
+			return nil, err
+		}
+		if storageSafetyStr != "" && storageSafetyStr != "optional" && storageSafetyStr != "encrypted" {
```

Comment:
couldn't this also be 

```suggestion
		if storageSafetyStr != "" && storageSafetyStr != ModelStorageSafetyOptional && storageSafetyStr != ModelStorageSafetyEncrypted {
```